vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -786,6 +791,11 @@ void WindowsTextInputComponentView::OnKeyUp(
       lParam |= 0x40000000; // bit 30
     lParam |= 0x80000000; // bit 31 always 1 for WM_KEYUP
 
+    // check to see if shift is released
+    if (args.Key() == winrt::Windows::System::VirtualKey::Shift) {
```

Comment:
this works but is there a better way to know if the shift key is pressed down? Didn't want to add the rest of the keys until I got confirmation on this :)

Reply:
You shouldn't try to keep track of the key state in WindowsTextInputComponentView.  Instead you should query the state as needed.

The KeyboardSource object has a method to get the key state of a key, you should pass that to the ShouldSubmit function.